packages/sqlite/src/index.ts

```diff
@@ -241,7 +241,12 @@ class SQLiteDriver extends Driver {
     return result
   }
 
-  async drop() {
+  async drop(table: string) {
+    const droppedTable = this.database.tables[table]
+    this.#run(`DROP TABLE ${escapeId(droppedTable)}`)
```

Comment:
```suggestion
    this.#run(`DROP TABLE ${escapeId(table)}`)
```

Obviously wrong here. `tables[table]` is a `Model`, not a string`.

Reply:
I'm very sorry, I was working on this in codespace and the type hints there are a mess, sorry for making such a mistake.